clustering_experiments.ipynb

```diff
@@ -20,9 +20,11 @@
     "import trecs.matrix_ops as mo\n",
```

Comment:
for user & item clusters - should we be concerned about the distribution sizes?

Reply:
is there a way to normalize this? maybe/prob not but thought i'd ask

Reply:
no, I thought about it because obviously it would be nicer if everything would be evenly distributed. An alternative would be something like hierarchical clustering but I would not go there now